source/onboard/connected-workspaces.rst

```diff
@@ -152,6 +152,13 @@ Once a connection is established between two Mattermost servers, system admins c
 
     The ability to create a direct or group message with remote users across connected workspaces is only available when the feature flag ``EnableSharedChannelsDMs`` is enabled. While this feature is in :ref:`Beta <manage/feature-labels:beta>`, there are some known issues that may impact reliability of direct message delivery across servers.
 
+Plugin component interaction
+-----------------------------
+
+From Mattermost v10.10, plugin interactions such as slash commands, interactive buttons, and other plugin-generated components aren't displayed or accessible in shared channels by default to ensure a consistent experience across different Mattermost instances. System admins can enable the ``EnableSharedChannelsPlugins`` feature flag to enable these plugin interactions in shared channels.
+
+    When plugin components are enabled in shared channels, ensure all connected Mattermost instances have the same plugins installed and configured to avoid inconsistent user experiences. Plugin behaviors may vary between instances if different plugin versions or configurations are used.
```

Comment:
```suggestion
When plugin components are enabled in shared channels, we recommend ensuring that all connected Mattermost instances have the same plugins installed and configured to avoid inconsistent user experiences. Plugin behaviors can vary between instances if different plugin versions or configurations are used.
```